src/test/scala/strawman/collection/test/TraverseTest.scala

```diff
@@ -69,9 +69,8 @@ class TraverseTest {
 
   @Test
   def eitherSequenceTest: Unit = {
-    val xs3 = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
+    val xs3: mutable.ListBuffer[Either[Int, String]] = mutable.ListBuffer(Right("foo"), Left(0), Right("bar"))
```

Comment:
If the collection strawman defined its own version of `Either` it could do the same by hand in the `apply` of `Left` and `Right`.